test/unit/api-client.spec.ts

```diff
@@ -1,332 +0,0 @@
-import { RevAiApiClient } from '../../src/api-client';
```

Comment:
This was supposed to be removed since this file was split into multiple but it got readded.